src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -28,5 +28,8 @@ public WatermarkOffsets(Offset low, Offset high)
 
         public Offset Low { get; }
         public Offset High { get; }
+
+        public override string ToString()
+            => $"[{Low}, {High}]";
```

Comment:
[L, H] is in fact correct mathematical notation for a closed interval (which this is), though mathematicians are not the audience i guess.

What about:

`<low> -> <high>`

?

I was just thinking we're being somewhat inconsistent with `ToString`. `Metadata` gives a JSON representation of the object (which will generally be quite large). Everything else something custom. I think this is what we want.